lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua

```diff
@@ -51,3 +51,12 @@ E2Helper.Descriptions["ejectPodTo(e:v)"]       = "Ejects driver of seat to desir
 -- Player Functions
 E2Helper.Descriptions["getAlive()"]            = "Returns an array with all alive players."
 E2Helper.Descriptions["getDead()"]             = "Returns an array with all dead players."
+
+-- NPC functions
+E2Helper.Descriptions["npcSetMaxHealth(e:n)"]          = "Sets an NPC's maximum health. Admin only."
+E2Helper.Descriptions["npcSetHealth(e:n)"]             = "Sets an NPC's health. Admin only."
+E2Helper.Descriptions["npcSetScale(e:n)"]              = "Sets the scale of an NPC. Admin only."
+E2Helper.Descriptions["npcSetDamageMultiplier(e:n)"]   = "Sets the damage multipler for an NPC, so if N = 2, an NPC that does 10 damage will do 20. Admin only"
```

Comment:
The name of the function accurately describes what it does, defining the word "multiplier" is unnecessary here.

```suggestion
E2Helper.Descriptions["npcSetDamageMultiplier(e:n)"]   = "Sets the damage multiplier for an NPC. Admin only."
```